src/test/java/com/kodilla/ecommercee/repository/ProductTestSuite.java

```diff
@@ -0,0 +1,191 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.*;
+import com.kodilla.ecommercee.repository.*;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.transaction.Transactional;
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+
+public class ProductTestSuite {
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private CartRepository cartRepository;
+
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @Test
+    public void productAddingTest() {
+
+        //Given
+        Product product1 = new Product("Shoes", "Super comfortable running shoes", new BigDecimal(199.99));
+        Product product2 = new Product("Suit", "100% wool business suit", new BigDecimal(1199.00));
+        Product product3 = new Product("Dress", "Posh evening dress", new BigDecimal(799.00));
+
+
+        //When
+        productRepository.save(product1);
+        productRepository.save(product2);
+        productRepository.save(product3);
+
+        List<Product> productList = (List<Product>) productRepository.findAll();
+
+        long productId = product1.getProductId();
+        String product1Name = product1.getProductName();
+
+        product3.setProductName("Evening Dress");
+        productRepository.save(product3);
+        String newProduct3Name = product3.getProductName();
+
+        //Then
+        assertEquals(3, productList.size());
+        assertEquals("Suit", productList.get(1).getProductName());
+        assertEquals("Shoes", product1Name);
+        assertEquals("Evening Dress", newProduct3Name);
+        assertNotEquals(0, productId);
```

Comment:
no i tu ... 135-137,  16-164, 188 też :)

Reply:
Faktycznie, teraz widzę. Ale w sumie to są testy, o których Hubert pisał, że są niepotrzebne bo testują nie to co trzeba w tej encji (zostawiłem je z sentymentu :) ). Może dla dobra realizacji projektu, odpuśćmy już ich naprawę tutaj (a sobie pokombinuję z nimi dla siebie).

Reply:
spoko, nie wiedziałem wcześniejszych uwag Huberta.